MANUAL.md

```diff
@@ -227,6 +230,21 @@ The `http_client` section defines RMT's global HTTP connection settings.
     Determines proxy authentication mechanism, possible values are:
     `none`, `basic`, `digest`, `gssnegotiate`, `ntlm`, `digest_ie`, `ntlm_wb`
 
+**Web server settings**
+
+The `web_server` section lets you tune the performance of the RMT server, by
+combining threads and workers to configure concurrency.
```

Comment:
```suggestion
The `web_server` section lets you tune the performance of your RMT server.
```

Not a blocking comment:

I don't think that we need to go too much into detail here.